helpers/data.js

```diff
@@ -10,3 +10,49 @@ export const data = {
     username: "jean786786"
   }
 };
+
```

Comment:
Can you please move this file into test/helpers/ as the sendGridResponse object is a test data?